gateware/vga/dma.py

```diff
@@ -0,0 +1,134 @@
+from migen.fhdl.std import *
```

Comment:
Should we add a comment about trying to share the DMA between the HDMI and VGA code? Something like;
```c
/* TODO: Investigate how to share the DMA code between HDMI (see gateware/hdmi/dma.py) and VGA code. See https://github.com/timvideos/HDMI2USB-misoc-firmware/issues/333. */
```